frontend/assets/stylesheets/style.scss

```diff
@@ -76,6 +76,7 @@
 @import 'components/nav/brand-bar';
 @import 'components/action';
 @import 'components/page';
+@import 'components/page-form';
```

Comment:
Not sure about the naming for this. Perhaps:

~~~ scss
@import 'components/form'; // Core form styles
@import 'components/form-layout'; // Form layouts
~~~